.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,10 +2,14 @@
 1. Make sure to read the Contributing Guide before submitting your PR: https://github.com/helm/helm/blob/master/CONTRIBUTING.md
 2. If this PR closes another issue, add 'closes #<issue number>' somewhere in the PR summary. GitHub will automatically close that issue when this PR gets merged. Alternatively, adding 'refs #<issue number>' will not close the issue, but help provide the reviewer more context.-->
 
+**Which issue(s) this PR fixes**:
+
 **What this PR does / why we need it**:
 
 **Special notes for your reviewer**:
 
+**Docs Changes**:
```

Comment:
This doesn't need to be here necessarily. If there are doc changes, it'll be part of the PR and can be noted as part of "what this PR does".

Reply:
> This doesn't need to be here necessarily. If there are doc changes, it'll be part of the PR and can be noted as part of "what this PR does".

Hmm, I think it will be more detail, such as [Kubernetes PR template](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

Reply:
That PR template does not include a note about "docs changes", so I'm not sure what you're trying to point me to look at. Can you please remove this for now? Thanks.